lib/functions.php

```diff
@@ -1658,6 +1658,7 @@ function newsletter_register_title_menu_items(ElggEntity $container_entity) {
 				'elgg-button-action',
 				'elgg-lightbox',
 			],
+			'data-colorbox-opts' => json_encode(['minWidth' => '500px','maxWidth' => '1000px']),
```

Comment:
I don't see a difference when I try this.
Also minWidth isn't supported by colorbox https://www.jacklmoore.com/colorbox/#setting-dimensions

Reply:
> I don't see a difference when I try this.
Try to open a newsletter form if group has a very loooong name.
My PR fix this issue.